### PR TITLE
Include `/dashboard-feed` in `isDashboard`

### LINKED ISSUE #186: Add `/dashboard-feed` to `isDashboard`?
See https://github.com/dashboard-feed

We're not removing the features just yet, https://github.com/refined-github/refined-github/pull/6887 so we might as well support this view for now.

 cc @port19x will be happy

### PR DESCRIPTION
**Pull request by port19x**
I've never written javascript or typescript, but if I don't grossly misunderstand the semantics here my PR should work.

fixes #186

### CODE REVIEW

**Comment by port19x** [comment]
Nice CI pipeline, got some fixes to do

**Comment by port19x** [comment]
I see, lockfile complains. Gotta install npm :walking:

File: index.ts [hunk 1]
```diff
@@ -81,6 +81,8 @@ addTests('isDashboard', [
 	'https://github.com/?tab=following', // Gotcha for `isUserProfileFollowingTab`
 	'https://github.com/?tab=overview', // Gotcha for `isUserProfileMainTab`
 	'https://github.com?search=1', // Gotcha for `isRepoTree`
+	'https://github.com/dashboard-feed',
+	'https://github.com//dashboard-feed',
```
Comment (fregante) [inline index.ts:85]:
```suggestion
```

Not necessary here

**Review by fregante — APPROVED** [review]
LGTM thanks!

**Comment by port19x** [comment]
Should I squash?

**Comment by fregante** [comment]
Ouch I forgot to publish this earlier 🥲 
It'll appear in the next version of Refined GitHub

**Pull request merged**